src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -169,47 +169,47 @@ class CompanyStep extends React.Component {
                                 />
                             </View>
                         </View>
-                        <TextInputWithLabel
+                        <ExpensiTextInput
                             label={this.props.translate('common.zip')}
                             containerStyles={[styles.mt4]}
                             onChangeText={addressZipCode => this.setState({addressZipCode})}
                             value={this.state.addressZipCode}
                         />
-                        <TextInputWithLabel
+                        <ExpensiTextInput
                             label={this.props.translate('common.phoneNumber')}
                             containerStyles={[styles.mt4]}
                             keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
                             onChangeText={companyPhone => this.setState({companyPhone})}
                             value={this.state.companyPhone}
                             placeholder={this.props.translate('companyStep.companyPhonePlaceholder')}
                         />
-                        <TextInputWithLabel
+                        <ExpensiTextInput
                             label={this.props.translate('companyStep.companyWebsite')}
                             containerStyles={[styles.mt4]}
                             onChangeText={website => this.setState({website})}
                             value={this.state.website}
                         />
-                        <TextInputWithLabel
+                        <ExpensiTextInput
                             label={this.props.translate('companyStep.taxIDNumber')}
                             containerStyles={[styles.mt4]}
                             keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
                             onChangeText={companyTaxID => this.setState({companyTaxID})}
                             value={this.state.companyTaxID}
                             disabled={shouldDisableCompanyTaxID}
                         />
-                        <Text style={[styles.formLabel, styles.mt4]}>
-                            {this.props.translate('companyStep.companyType')}
-                        </Text>
-                        <Picker
-                            items={_.map(CONST.INCORPORATION_TYPES, (label, value) => ({value, label}))}
-                            onChange={incorporationType => this.setState({incorporationType})}
-                            value={this.state.incorporationType}
-                            placeholder={{value: '', label: 'Type'}}
-                        />
+                        <View style={styles.mt4}>
+                            <Picker
```

Comment:
Is there a reason you kept `Picker` here and didn't use `ExpensiPicker?`

Reply:
I've must been missed it) Implementing now